server/sonar-web/src/main/js/apps/projects/components/ProjectCard.js

```diff
@@ -35,7 +36,10 @@ export default class ProjectCard extends React.PureComponent {
     project?: {
       analysisDate?: string,
       key: string,
-      name: string
+      name: string,
+      isFavorite?: boolean,
+      organization?: string,
+      tags?: Array<string>
```

Comment:
In the end I didn't drop the `?` because for the moment we have a case where the `tags` doesn't exists. It's when I click on a project without tags to see its detail and then come back to the projects page.